src/parser/view_transform.c

```diff
@@ -11717,9 +11732,16 @@ mq_fetch_subqueries_for_update_local (PARSER_CONTEXT * parser, PT_NODE * class_,
       if (!query_cache->view_cache->vquery_for_update
 	  && (!query_cache->view_cache->vquery_for_partial_update || (fetch_as != PT_PARTIAL_SELECT)) && parser)
 	{
-	  PT_ERRORmf (parser, class_, MSGCAT_SET_PARSER_RUNTIME, MSGCAT_RUNTIME_VCLASS_NOT_UPDATABLE,
-		      /* use function to get name. class_->info.name.original is not always set. */
-		      db_get_class_name (class_object));
+	  char reuse_tbl[300] = "";
+
+	  if (reuse_oid_class_name)
+	    {
+	      sprintf (reuse_tbl, ", because %s table is REUSE_OID.", reuse_oid_class_name);
```

Comment:
The period is redundant.
```suggestion
	      sprintf (reuse_tbl, ", because %s table is REUSE_OID", reuse_oid_class_name);
```

Reply:
Ok, I will fix it.